src/amo/pages/AddonInfo/styles.scss

```
@@ -18,3 +18,15 @@
     width: 65%;
   }
 }
+
+.AddonMoreInfo-tag-links-list > li {
+  display: inline-block;
+}
+
+.AddonMoreInfo-tag-links-list > li::after {
+  content: ',\00a0';
+}
+
+.AddonMoreInfo-tag-links-list > li:last-child::after {
+  content: '';
+}
```

We usually avoid `>` and it is sass so it can be written like this:


```suggestion
.AddonMoreInfo-tag-links-list {
  li {
    display: inline-block;
  }

  li::after {
    content: ',\00a0';
  }

  li:last-child::after {
    content: '';
  }
}
```